pkg/sql/distsql_physical_planner.go

```diff
@@ -1036,6 +1036,9 @@ func (l *DistLoader) LoadCSV(
 	if err != nil {
 		return err
 	}
+	if log.V(1) {
+		log.Infof(ctx, "import-distsql: begin sampling phase of job %s", job.Record.Description)
```

Comment:
I would tag the context `log.WithLogTag` to not have to write `import-distsql` every time.